.php-cs-fixer.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$config = \TYPO3\CodingStandards\CsFixerConfig::create();
+$config->getFinder()->in('Classes');
```

Comment:
To check the same files as before, we'll also need to check `Tests/` and `Configuration/`.